CougSat1-ADCS/src/interfaces/CDH.h

```diff
@@ -2,16 +2,21 @@
 #define _SRC_INTERFACES_CDH_H_
 
 #include <mbed.h>
+#include <rtos.h>
+#include "Configuration.h"
 
 class CDH {
 public:
-  CDH(I2CSlave & i2c);
-
-  bool              hasMessage();
-  mbed_error_code_t processMessage();
+  CDH(uint8_t addr, PinName sda, PinName scl);
+  void readCDH();
```

Comment:
Don't need to say readCDH. As when you call the function cdh.readCDH, cdh is redundant.